service-app/internal/factory/document_component.go

```diff
@@ -51,6 +53,14 @@ func GetComponent(docType string) (Component, error) {
 			Validator: lpc_parser.NewValidator(),
 			Sanitizer: lpc_parser.NewSanitizer(),
 		}, nil
+	case constants.DocumentTypeLPA002, constants.DocumentTypeLPAPA, constants.DocumentTypeLPAPW, constants.DocumentTypeLPA114, constants.DocumentTypeLPA117:
```

Comment:
I know this is a POC, but just to mention tthe approach has changed with https://github.com/ministryofjustice/opg-scanning/pull/50/files